### PR TITLE
[TASK] Update psr/log dependency

### LINKED ISSUE #112: version conflict with Symfony and psr/log
symfony 2.3 requires `"psr/log": "~1.0"` and `1.0.1` is the latest version, but shariff requires `"psr/log": "1.0.0"`

this broke our deployment yesterday.

img_2016-09-23 11 24 38

we currently force `"psr/log": "1.0.0"` so that we can continue deployment but it would be nice to have following updates of `"psr/log"`

### PR DESCRIPTION
**Pull request by liayn**
Fixes #112

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "require": {
     	"php": ">=5.5.0",
         "guzzlehttp/guzzle": "^6.1",
-        "psr/log": "1.0.0",
+        "psr/log": "~1.0",
```
Comment (core23) [inline composer.json]: Please use the ^1.0 notation

Comment (liayn) [inline composer.json]: This makes absolutely no difference in that case. But I'll push a new version for you ;-)

Comment (core23) [inline composer.json]: `~1.0` and `^1.0` are the same, but it would it would conveniente to have the same notation for all dependencies.

**Comment by compeak** [comment]
Thanks.

**Pull request merged**